hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
So I've made an edit this morning. Now the user can set beam_area in the metadata. If they don't set a beam_area the function will then use the scaling from the meta.data. This should allow line scans to be possible again if they set the beam_area. I've also added more discussion about beam_area vs pixel_area in the documentation. Hopefully that is more accessible now.